tests/arch/x86/cpu_scrubs_regs/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+ arch.x86.syscalls.cpu_scrubs_regs:
+    filter: CONFIG_ARCH_HAS_USERSPACE
```

Comment:
Need to filter on x86 arch

Reply:
@MaureenHelm I added `arch_allow: x86` and removed filter